Source/Core/Core/PowerPC/Jit64/Jit_LoadStore.cpp

```diff
@@ -474,18 +474,9 @@ void Jit64::dcbz(UGeckoInstruction inst)
     FixupBranch slow = J_CC(CC_Z, Jump::Near);
 
     // Fast path: compute full address, then zero out 32 bytes of memory.
-    if (cpu_info.bAVX)
-    {
-      VXORPS(XMM0, XMM0, R(XMM0));
-      VMOVAPS(MComplex(RMEM, RSCRATCH, SCALE_1, 0), YMM0);
-      VZEROUPPER();
-    }
-    else
-    {
-      XORPS(XMM0, R(XMM0));
-      MOVAPS(MComplex(RMEM, RSCRATCH, SCALE_1, 16), XMM0);
-      MOVAPS(MComplex(RMEM, RSCRATCH, SCALE_1, 0), XMM0);
-    }
+    XORPS(XMM0, R(XMM0));
+    MOVAPS(MComplex(RMEM, RSCRATCH, SCALE_1, 0), XMM0);
+    MOVAPS(MComplex(RMEM, RSCRATCH, SCALE_1, 16), XMM0);
```

Comment:
I know nothing about Jit or assembly stuff.  However, this is basically a revert to the previous logic.  I assume the `V` variants are avx specific, do they provide any speed boost on their own?

If so, maybe we could mimic the current logic but with the `V` variants?

Reply:
This is indeed a partial revert. I also don't know enough about x64 to know what the proper optimization is; my goal with this PR is just to fix the build and minimize the amount of time the latest Dev version is broken.